src/webview/serverless-manage-repository/app/showRepositories.tsx

```diff
@@ -0,0 +1,357 @@
+            openedRepo: '',
+            newRepoInput: {
+                name: '',
+                error: false,
+                helpText: ''
+            }
+        });
+    }
+
+    setRepoName = (value: string): boolean => {
+        this.setState({
+            openEditDialog: true,
+            newRepoInput: {
+                name: value,
+                error: false,
+                helpText: ''
+            }
+        });
+        return this.state.openEditDialog;
+    }
+
+    validateName = (value: string): void => {
+        VSCodeMessage.postMessage({
+            action: `validateNewName`,
+            data: value
+        });
+    }
+
+    handleDisable = (): boolean => {
+        return this.state.newRepoInput.name.length === 0 || this.state.newRepoInput.error
+    }
+
+    rename(oldName: string, newName: string): void {
+        this.CloseDialog();
+        VSCodeMessage.postMessage({
+            action: `renameRepo`,
+            data: {
+                oldName,
+                newName
+            }
+        });
+    }
+
+    StyledTableCell = styled(TableCell)(({ theme }) => ({
+        [`&.${tableCellClasses.head}`]: {
+            backgroundColor: 'var(--vscode-button-background)',
+            color: 'var(--vscode-button-foreground)'
+        },
+        [`&.${tableCellClasses.body}`]: {
+            fontSize: 14,
+        },
+    }));
+
+    StyledTableRow = styled(TableRow)(({ }) => ({
+        '&:nth-of-type(odd)': {
+            backgroundColor: 'var(--vscode-button-secondaryBackground)'
+        },
+
+        '&:nth-of-type(even)': {
+            backgroundColor: 'var(--vscode-button-secondaryHoverBackground)'
+        }
+    }));
+
+    handleChangeRowsPerPage = (event: React.ChangeEvent<HTMLInputElement>) => {
+        this.setState({
+            page: 0,
+            rowsPerPage: parseInt(event.target.value, 10)
+        });
+    };
+
+    handleChangePage = (_event: unknown, newPage: number) => {
+        this.setState({
+            page: newPage
+        });
+    };
+
+    render(): React.ReactNode {
+        const { newRepoInput, openedRepo, openAddDialog, openDeleteDialog, openEditDialog, page, repositories, rowsPerPage } = this.state;
+        return (
+            <div className='mainContainer margin'>
+                <div className='title'>
+                    <Typography variant='h5'>Manage Repositories</Typography>
+                </div>
+                <div className='subTitle'>
+                    <Typography>Manage template repositories installed on disk at either the default location (~/.config/func/repositories) or the location specified by the --repository flag. Once added, a template from the repository can be used when creating a new function.</Typography>
+                </div>
+                <Container maxWidth='sm' sx={{
+                    border: '1px groove var(--vscode-activityBar-activeBorder)',
+                    borderRadius: '1rem', backgroundColor: '#101418'
```

Comment:
This colour doesn't adapt to the theme if you switch to light mode. See https://github.com/redhat-developer/vscode-openshift-tools/commit/40008f4e04c2109ee97f72d8b1c9a6dc90626db1 for a possible fix.